state-chain/runtime/src/runtime_apis.rs

```diff
@@ -447,6 +508,41 @@ pub struct NetworkFees {
 	pub internal_swap_network_fee: NetworkFeeDetails,
 }
 
+#[derive(Encode, Decode, TypeInfo, Serialize, Deserialize, Clone, Default)]
+pub struct RpcAccountInfoCommonItems<Balance> {
+	pub flip_balance: Balance,
+	pub asset_balances: cf_chains::assets::any::AssetMap<Balance>,
+	pub bond: Balance,
+	pub estimated_redeemable_balance: Balance,
+	#[serde(skip_serializing_if = "Option::is_none")]
+	pub bound_redeem_address: Option<EthereumAddress>,
+	#[serde(skip_serializing_if = "BTreeMap::is_empty")]
+	pub restricted_balances: BTreeMap<EthereumAddress, Balance>,
+	#[serde(skip_serializing_if = "Option::is_none")]
+	pub delegating_to: Option<AccountId32>,
```

Comment:
maybe this can be an optional struct and contain info about the :

```suggestion
	pub delegating_to: Option<DelegationInfo>,
	pub delegating_to_next: Option<DelegationInfo>,
```

```rs
struct DelegationInfo {
    delegating_to: AccountId32,
    bid: Balance,
}
```

`delegating_to` captures the info from the current epoch, `delegating_to_next` (or whatever better name there is) captures the current delegation info, i.e. if the auction resolves now, this will be their new delegation for the upcoming epoch